dev/WindowsAppRuntime_MSIXInstallFromPath/main.cpp

```diff
@@ -161,29 +169,37 @@ HRESULT JustDoIt(PCWSTR path) noexcept try
         auto packageFullName{ std::wstring(line.c_str() + offset + 1) };
         THROW_HR_IF_MSG(E_UNEXPECTED, packageFullName.empty(), "line:%s", lineUtf8.c_str());
 
-        AddPackageIfNecessary(path, filename, packageFullName);
+        AddPackageIfNecessary(path, filename, packageFullName, forceDeployment);
+    }
+
+    // Restart Push Notifications Long Running Platform when ForceDeployment option is applied.
+    if (forceDeployment)
+    {
+        wil::com_ptr_nothrow<INotificationsLongRunningPlatform> longRunningProcessPlatform{
+            wil::CoCreateInstance<NotificationsLongRunningPlatform, INotificationsLongRunningPlatform>(CLSCTX_LOCAL_SERVER) };
```

Comment:
Same concerns here. I think it would be nice to log and track errors here if Activation fails for any reason.